crates/bevy_app/src/schedule_runner.rs

```diff
@@ -6,9 +6,11 @@ use crate::{
 use bevy_utils::{Duration, Instant};
 
 #[cfg(target_arch = "wasm32")]
-use std::{cell::RefCell, rc::Rc};
-#[cfg(target_arch = "wasm32")]
-use wasm_bindgen::{prelude::*, JsCast};
+use {
+    alloc::rc::Rc,
+    core::cell::RefCell,
+    wasm_bindgen::{prelude::*, JsCast},
+};
```

Comment:
Due to how `cargo +nightly fmt` split use statements, there were many instances of duplicated conditional compilation attributes. As a part of re-merging imports, I grouped these conditional imports together. I think this is easier to read, but may be contentious.